packages/suite/src/components/suite/SecurityCheck/SecurityCheckLayout.tsx

```diff
@@ -52,12 +53,31 @@ export const SecurityCheckLayout = ({ isFailed, children }: SecurityCheckLayoutP
 
     const deviceModelInternal = device?.features?.internal_model;
     const imageVariant = isFailed ? 'GHOST' : 'LARGE';
+    const isDeviceImageRotating =
+        deviceModelInternal &&
+        [DeviceModelInternal.T1B1, DeviceModelInternal.T3B1, DeviceModelInternal.T3T1, DeviceModelInternal.T2T1, DeviceModelInternal.T2B1].includes(
+            deviceModelInternal,
+        );
 
-    return (
+    // @ts-expect-error: this code doesn't throw but must be deleted as it is for debugging
+    const searchParams = new URLSearchParams(location.search);
+
+    return (    
         <Wrapper>
             {deviceModelInternal && (
                 <ImageWrapper>
-                    <StyledImage image={`TREZOR_${deviceModelInternal}_${imageVariant}`} />
+                    {isDeviceImageRotating ? (
+// @ts-expect-error: this cannot be here as SecurityCheckLayout is used in multiple places a different API must be made for it.
```

Comment:
@jvaclavik How do we do this right? Basically, there's a "Layout" component that is used everywhere and just automatically uses the "fixed" images.

A simple boolean prop indicating that we want a rotating version of the illustration image may be enough.